packages/serverless-components/nextjs-cdk-construct/__tests__/fixtures/next-boilerplate/default-lambda/index.js

```diff
@@ -0,0 +1 @@
+exports.handler = async () => ({ statusCode: 200 });
```

Comment:
For better readability of all these added JS/JSON files, could you please format these test files using prettier?

Reply:
Done in https://github.com/serverless-nextjs/serverless-next.js/pull/878/commits/ea6cc1605efab672a2aa586fe43cd3de8fec38d2

Reply:
Although, prettier has not updated `__tests__/fixtures/next-boilerplate/default-lambda/index.js`...